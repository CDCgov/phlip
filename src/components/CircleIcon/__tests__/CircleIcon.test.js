import React from 'react'
import { shallow } from 'enzyme'
import { CircleIcon } from '../index'

const props = {
  circleColor: 'error',
  iconColor: '#fff',
  iconSize: 18,
  circleSize: 32,
  children: 'home',
  theme: {
    palette: {
      error: {
        500: '#ff3d70'
      }
    }
  }
}

describe('CircleIcon component', () => {
  test('should render correctly', () => {
    expect(shallow(<CircleIcon {...props} />)).toMatchSnapshot()
  })
})