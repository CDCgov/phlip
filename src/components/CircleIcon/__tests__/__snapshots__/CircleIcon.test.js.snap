// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircleIcon component should render correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ff3d70",
      "borderRadius": "50%",
      "display": "flex",
      "height": 32,
      "width": 32,
    }
  }
>
  <WithTheme(Icon)
    color="#fff"
    size={18}
    style={
      Object {
        "flex": "1",
        "textAlign": "center",
      }
    }
  >
    home
  </WithTheme(Icon)>
</div>
`;
